native/level1double.go

```diff
@@ -127,6 +127,7 @@ func (Implementation) Idamax(n int, x []float64, incX int) int {
 	idx := 0
 	max := math.Abs(x[0])
 	if incX == 1 {
+		x = x[:n]
```

Comment:
Do this in the range statement `range x[:n]`?; x should not used after this (except that it is again in the explicit for loop below - why?)